extensions.json

```
@@ -1235,6 +1235,10 @@
       "id": "travis.simple-dark",
       "repository": "https://github.com/HondryTravis/simple-dark"
     },
+    {
+      "id": "tuxtina.json2yaml",
+      "repository": "https://github.com/tuxtina/json2yaml"
```

When available, we prefer re-publishing official `.vsix` releases instead of trying to repackage a release ourselves.

You can achieve the result below by using this command: `node add-extension --download=https://github.com/tuxtina/json2yaml/releases/download/v0.2.0/json2yaml-0.2.0.vsix`:

```suggestion
      "download": "https://github.com/tuxtina/json2yaml/releases/download/v0.2.0/json2yaml-0.2.0.vsix",
      "version": "0.2.0"
```

---

Thanks for the fix :+1: